.github/workflows/installers-conda.yml

```diff
@@ -234,26 +249,32 @@ jobs:
 
           installer -pkg $PKG_PATH -target CurrentUserHomeDirectory >/dev/null
 
-          root_prefix=$(compgen -G $HOME/Library/spyder-*)
+          base_prefix=$(compgen -G $HOME/Library/spyder-*)
+          spy_rt=$base_prefix/envs/spyder-runtime
+          menu=$spy_rt/Menu/spyder-menu.json
+          mode=user
+          shortcut=$(${base_prefix}/bin/python - <<EOF
+          from menuinst.api import _load
+          menu, menu_items = _load("$menu", target_prefix="$spy_rt", base_prefix="$base_prefix", _mode="$mode")
+          print(menu_items[0]._paths()[0])
+          EOF
+          )
```

Comment:
Perhaps it'd be a good idea to move this code to a Python script that prints the shortcut location and has two cli options: `--base-prefix` and `--mode` (if I understand things correctly, the other variables can be composed from those two).

That way we would only need to call that script here, which would make things more readable and maintainable. And it could also be used to test the Linux installer.

If you think this is a good idea, you could place that script in `.github/scripts` and call it (for instance) `get_installer_shortcut.py`

Reply:
Good idea.